### PR TITLE
Make index management more reachable [doc only]

### PR DESCRIPTION
**Pull request by farodin91**
Part of #460

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by farodin91** [comment]
Preview https://farodin91.github.io/janusgraph/

File: docs/index-management/index-reindexing.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# Index Reindexing
```
Comment (FlorianHockmann) [inline docs/index-management/index-reindexing.md]: _Index Reindexing_ sounds strange. Why not just _Reindexing_?

File: docs/index-management/index-common-problems.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Common Problems with Index Management
+
+## IllegalArgumentException when starting job
+
+When a reindexing job is started shortly after a the index has been
```
Comment (FlorianHockmann) [inline docs/index-management/index-common-problems.md]: These problems are specific to reindexing and index removal, but they now stand independent of that which is confusing in my opinion.
I'm not sure if it's really a good idea for that reason to split the current _Index Management_ page into three different pages as they are about the same topic, namely managing already existing indices.

File: docs/index-management/index-reindexing.md (hunk 1 above)
Comment (farodin91) [reply docs/index-management/index-reindexing.md]: If you only write reindexing, what do we want to reindex, e.g. a file system.

File: docs/index-management/index-common-problems.md (hunk 2 above)
Comment (farodin91) [reply docs/index-management/index-common-problems.md]: It currently only contains reindex and removal part. I see a lot of question to all of these sections.

The hole section is called Index Management. Index management should also contain creation.

**Pull request merged**